app/views/welcome/index.html.erb

```diff
@@ -3,152 +3,159 @@
-                </div>
-
-                <div class="col-md-6" id="browse-issues-button">
-                    <%= link_to "Browse 2016 Issues", issues_path, class: "btn btn-lg btn-default btn-block" %>
-                </div>
-            </div>
-        </div>
-
-        <div class="well">
-            <div class="row">
-                <div class="col-md-6a" id="login-button">
-                    <%= link_to "Login", login_path, class: "btn btn-lg btn-default btn-block" %>
-                </div>
-            </div>
-        </div>
-
-        <hr>
-
-        <!-- Footer -->
-        <footer>
-            <div class="row">
-                <div class="col-lg-12">
-                    <p>Copyright &copy; Earmarked 2015</p>
-                </div>
-            </div>
-        </footer>
-
-    </div>
-    <!-- /.container -->
-
-    <!-- jQuery -->
-    <script src="js/jquery.js"></script>
-
-    <!-- Bootstrap Core JavaScript -->
-    <script src="js/bootstrap.min.js"></script>
-
-    <!-- Script to Activate the Carousel -->
-    <script>
-    $('.carousel').carousel({
-        interval: 5000 //changes the speed
-    })
-    </script>
+		<!-- Header Carousel -->
+		<header id="myCarousel" class="carousel slide">
+				<!-- Indicators -->
+				<ol class="carousel-indicators">
+						<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+						<li data-target="#myCarousel" data-slide-to="1"></li>
+						<li data-target="#myCarousel" data-slide-to="2"></li>
+				</ol>
+
+				<!-- Wrapper for slides -->
+				<div class="carousel-inner">
+						<div class="item active">
+								<div class="fill" id="slider-1">
+					 			</div>
+						</div>
+						<div class="item">
+								<div class="fill" id="slider-2"></div>
+						</div>
+						<div class="item" >
+								<div class="fill" id="slider-3"></div>
+						</div>
+				</div>
+
+				<!-- Controls -->
+				<a class="left carousel-control" href="#myCarousel" data-slide="prev">
+						<span class="icon-prev"></span>
+				</a>
+				<a class="right carousel-control" href="#myCarousel" data-slide="next">
+						<span class="icon-next"></span>
+				</a>
+		</header>
+
+		<!-- Page Content -->
+		<div class="container">
+
+				<!-- Marketing Icons Section -->
+				<div class="row">
+						<div class="col-lg-12">
+								<h1 class="page-header" id='title-1'>
+									Welcome to Earmarked
+								</h1>
+						</div>
+					</div>
+
+				<!-- /.row -->
+
+				<!-- Candidate Section -->
+				<div class="row">
+						<div class="col-lg-12">
+								<h2 class="page-header" id="candidates"> Top Candidates</h2>
+						</div>
+						<% @featured_candidates.each do |candidate| %>
+						<div class="col-md-4 col-sm-6">
+								<a href="portfolio-item.html">
+										<%= link_to image_tag("#{candidate.last_name}.jpg", class:"img-responsive img-portfolio img-hover"), candidate_path(candidate.id)%>
```

Comment:
Correct - based on the new column I made "last_name". I am interpolating through each candidate and calling the associated .jpg. But the way we had it stored "Donald Trump".jpg would not work so I had to make a column I could reference to pull out the images.